src/page-response-impact/rtStatistics.tsx

```diff
@@ -0,0 +1,25 @@
+import { mean } from "lodash";
+
+import { getNewestRt, getOldestRt, RtRecord } from "../infection-model/rt";
+
+export function numFacilitiesWithRtLessThan1(
+  facilitiesRtRecords: RtRecord[][],
+) {
+  const latestRtValues = facilitiesRtRecords.map((rtRecords) => {
+    return getNewestRt(rtRecords)?.value;
+  });
+  return latestRtValues.filter((v) => v && v < 1).length;
```

Comment:
`v` could be `0` here, right? in which case we would actually want to include it in the count

Reply:
Oh yeah good catch, I always forget that 0 evaluates to false.